libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -163,9 +192,24 @@ - (BOOL)idpEnabled {
 }
 
 - (void)setIdpEnabled:(BOOL)idpEnabled {
+    if (idpEnabled) {
```

Comment:
You need an `else` block too that calls `[SFSDKAppFeatureMarkers removeAppFeature:kSFSPAppFeatureIDPLogin];`, since the flag can be toggled the other way and we'd have to remove the existing app marker then. This can happen when a new value is pushed down through MDM that enables/disables this feature.